### DISCUSSION TITLE
"No implementation was created" error

### DISCUSSION THREAD

**Discussion by maximkr** [mapstruct/mapstruct#3533] [Q&A]
Hello!

I get the following error messages when trying to build a project. I don't see anything strange in my sources. I have verbose=true enabled, but I didn't see any details in the log. How to debug this problem?

I cannot post source code because it's a large project, I've just extend it trivial way in many places.

```
error: No implementation was created for NotificationMapper due to having a problem in the erroneous element java.util.ArrayList. Hint: this often means that some other annotation processor was supposed to process the erroneous element. You can also enable MapStruct verbose mode by setting -Amapstruct.verbose=true as a compilation argument.

error: No implementation was created for AlternativeAppConfirmMapper due to having a problem in the erroneous element java.util.ArrayList. Hint: this often means that some other annotation processor was supposed to process the erroneous element. You can also enable MapStruct verbose mode by setting -Amapstruct.verbose=true as a compilation argument.
```

**Comment by thunderhook** [comment]
Hard to help without code.
But this seems to be related to the setup of the annotation processor itself. Have a look at:
* https://github.com/mapstruct/mapstruct/issues/3124#issue-1511112111

Maybe you can post the code of your pom/gradle file?
